src.common/Enums/Direction8.ts

```diff
@@ -1,14 +1,35 @@
-export interface Direction8 {
-	readonly id: number;
-	readonly name: string;
-	readonly x: number;
-	readonly y: number;
-	readonly nextCW: Direction8;
-	readonly nextCCW: Direction8;
-	readonly opposite: Direction8;
-}
+export class Direction8 implements Direction8 {
```

Comment:
This class now implements ... itself? Is this still necessary?

Reply:
Whoopsie, missed it